### ISSUE TITLE
MYQ Lite garage door opener just stopped working this week....   dont know why

### ISSUE THREAD

**Issue by fdelima1** [brbeaird/SmartThings_MyQ#61]
Loaded the device driver and smart app months ago and ll was working fine..

Just realized that things stopped working this week..  Also noticed that Chamberline released a new app - saying that their old app is obsolete....   Is this simply a coincidence ?  

any ideas .. ive looked at github and im all upto date

**Comment by fkenisky** [comment]
I've been following these threads for a while now and couldn't figure out how to post.  Finally!  I've got a few questions now that My-Q has updated it's app.  With the older app I was able to load code from here to allow Alexa to access my garage door.  It would close it but I don't think it would open it, as I needed the app to do that.

No problems as I set up a routine so that my garage door was checked to be closed by 9:30pm every evening.  If it was closed then no problem.

I haven't had the chance to check if the routine still works.

My questions are:
1) Is the new My-Q app still work with the code?
2) I checked my Alexa and didn't find the skill located there any longer.  Is there a new skill?

Frank

**Comment by Mythris29** [comment]
I'm having the same issue. The updated app came out so I migrated to it and now my garage door isn't working with smarthings.

**Comment by brbeaird** [comment]
Did you guys update the SmartApp here? I just pushed out a fix a couple days ago - try that and see if it helps you.

**Comment by fdelima1** [comment]
Brian 

I don’t see anything in the ide when I update the device drivers or smart apps.  

Is there a way to determine if I have your update.  . 

Also I read that chamberlain stopped the port access from smartthings , so
Very curious how you got around it.   I’m Sure it’s genius....


Flavian de Lima, PMP®
IBM Executive, Delivery, Travel and Transportation Industry,
Certified Executive and Complex PM
IBM Global Business Services
Mobile - 647-225-2596
Toronto Office - 416-478-9779

Assistant : Celia Hernandez Vargas - 404-527-8260 x8844

**Comment by brbeaird** [comment]
If you just copy and paste the newest code from the SmartApp in github to
your IDE, you can scroll down a bit and see version 2.1.11. Then click save
and publish.

**Comment by Mythris29** [comment]
I could have sworn I checked community apps before I posted here and there wasn't an update...I must have been seeing things though because I did see an update this time that I checked and once it was applied it's all working again! 

Thank you for your work on this app!

**Comment by snowyogi** [comment]
The community installer is installing an old version. My door stopped working as well. I tried re-installing with the community installer, but it is installing an old version (.9). I resolved the problem by installing from the repo which is .11

The response I got in the app is "username and password is incorrect"

9f097920-c286-4534-8bec-66b9dafe1081 1:42:52 PM: debug doorVersion: 2.2.4
9f097920-c286-4534-8bec-66b9dafe1081 1:42:52 PM: debug latestVersion: 2.1.11
9f097920-c286-4534-8bec-66b9dafe1081 1:42:52 PM: debug installedVersion: 2.1.9
9f097920-c286-4534-8bec-66b9dafe1081 1:42:52 PM: debug previousVersion: 2.1.9
9f097920-c286-4534-8bec-66b9dafe1081 1:42:51 PM: warn API POST Error: groovyx.net.http.ResponseParseException: OK
9f097920-c286-4534-8bec-66b9dafe1081 1:42:51 PM: trace Logging in

**Comment by cptthermo** [comment]
Hi everyone!  I'm also having an issue.  I can still tell Alexa to turn garage door opener off, which closes the garage.  However, I can no longer open it.  When I arrive to my home, it does not trigger to open.  I have validated every file and they all match the most current versions and I also double checked that i'm on 2.1.11.  Anything else I should check?  This app has worked like a CHAMP for about a year!

**Comment by brbeaird** [comment]
Aside from Alexa, does it work directly from the SmartThings app? If not, you might try opening the live logging in the IDE when you send an opening command and see if there are any errors.

I can't say I've heard of someone seeing one command work but not the other.

**Comment by gffounder** [comment]
Hey Brian, have you seen any other apps here that integrate the non LITE version of MyQ with SmartThings?

**Comment by brbeaird** [comment]
> have you seen any other apps here that integrate the non LITE version of MyQ with SmartThings?

No. Any other approach requires constantly polling the MyQ API to get the status of the door, and MyQ does not like us doing that from SmartThings servers.

There are other projects in GitHub where you could set up your own server locally that would do the polling, but I'm not sure if any of those then integrate with SmartThings.
